components/server/src/server.rs

```diff
@@ -1016,9 +1017,16 @@ impl<ER: RaftEngine> TiKVServer<ER> {
     }
 
     fn init_storage_stats_task(&self, engines: Engines<RocksEngine, ER>) {
+        // currently only for RocksEngine.
+        if self.config.raft_engine.enable {
```

Comment:
It's wrong. raft_engine is only for replacing raftdb. rocksdb is still used even if raft_engine is enabled.

Reply:
right, forget to remove this after add the get_engine_size() fn.